src/tfwebserver/processors/link.cr

```diff
@@ -0,0 +1,59 @@
+require "markd"
+
+module TFWeb
+  class LinkProcessor < ContentProcessor
```

Comment:
I think we maybe can check the linksimagesfixer, and abstract the approach so we already parse the elements we are interested in 1 place instead?